public/index.js

```diff
@@ -1,6 +1,8 @@
 $(document).ready(function(){
-  $('.carousel').carousel({
-    indicators: true,
-    shift: 100
-  });
+  if ($('.carousel')) {
```

Comment:
Again: https://github.com/CYPIAPT-LNDSE/welcome-to-camhs/issues/58.

See my comment about unifying the way your js files are wrapped up (This file is wrapped in jquery.ready but the balloon.js is not)